modules/plottinggl/src/plotters/persistencediagramplotgl.cpp

```diff
@@ -70,7 +70,8 @@ PersistenceDiagramPlotGL::Properties::Properties(std::string identifier, std::st
                   vec4(1), vec4(0.1f), InvalidationLevel::InvalidOutput, PropertySemantics::Color)
     , lineColor_("lineColor", "Line Color", vec4(vec3(34, 96, 150) / 255.0f, 1.0f), vec4(0),
                  vec4(1), vec4(0.1f), InvalidationLevel::InvalidOutput, PropertySemantics::Color)
-    , hoverColor_("hoverColor", "Hover color", vec4(1.0f, 0.77f, 0.25f, 1))
+    , hoverColor_("hoverColor", "Hover Color", vec4(1.0f, 0.906f, 0.612f, 1))
```

Comment:
Future PR: Define common default colors and/or color schemes for hover and selection so that we don't need to copy them to each processor

Reply:
good idea. Maybe even as System settings. However, which module should be responsible for setting those up?

Reply:
Probably not since there might be cases where it is not desirable to have the same. Default linking would probably better

Reply:
Linking is not the same as default colors.